test/extended/scheduling/pods.go

```diff
@@ -89,33 +87,6 @@ func (p requirePodsOnDifferentNodesTest) getDeploymentPods(oc *exutil.CLI) []*co
 }
 
 func (p requirePodsOnDifferentNodesTest) run(oc *exutil.CLI) {
-	ctx := context.TODO()
```

Comment:
Do we have upgrade tests where this might be running?

Reply:
Only two jobs I could find:
- ./release/openshift-release-master__nightly-4.12-upgrade-from-stable-4.10.yaml
- ./release/openshift-release-master__ci-4.12-upgrade-from-stable-4.11-from-stable-4.10.yaml

Checking the latter openshift-tests built from 4.12/master is run inside the job.

Reply:
https://github.com/openshift/origin/pull/26899 was merged in 4.11/master. Meaning 4.10 should comply with the test as well.